### PR TITLE
Fix "should be equal" for boolean values + Add "should not be equal to"

### PR DESCRIPTION
**Pull request by jdeniau**
There was an error with the "should be equal to" when the actual value is a boolean:
I added a test and fixed it.

I added a "should **not** be equal to" assertion.

Thank you !

### CODE REVIEW

**Comment by jdeniau** [comment]
I saw the https://github.com/Behatch/contexts/pull/115 PR, if this branch is merged before, I will update my commit to keep only the "should not be equal to" code.

File: src/Context/JsonContext.php [hunk 1]
```diff
@@ -56,6 +56,10 @@ public function theJsonNodeShouldBeEqualTo($node, $text)
 
         $actual = $this->inspector->evaluate($json, $node);
 
+        if (is_bool($actual)) {
+            $text = $text == 'true' ? true : false;
+        }
+
```
Comment (MLKiiwy) [inline src/Context/JsonContext.php]: Also add to test "null" value :
```php
if ($text === 'null') {
    $text = null
}
```

File: src/Context/JsonContext.php [hunk 2]
```diff
@@ -76,6 +80,28 @@ public function theJsonNodesShoudBeEqualTo(TableNode $nodes)
     }
 
     /**
+     * Checks, that given JSON node is not equal to given value
+     *
+     * @Then the JSON node :node should not be equal to :text
+     */
+    public function theJsonNodeShouldNotBeEqualTo($node, $text)
+    {
+        $json = $this->getJson();
```
Comment (MLKiiwy) [inline src/Context/JsonContext.php]: Could you factorise ? With the other sentence with a protected or private common function

**Comment by MLKiiwy** [comment]
Maybe rebase + check travis fix

**Comment by MLKiiwy** [comment]
Else great job :)

**Comment by MLKiiwy** [comment]
... great job but #116 do the job so maybe close this PR !

**Comment by sanpii** [comment]
#116 merged.
